app/views/movies/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1><%= @movie.title %> Show Page </h1>
+<p><%=  @movie.creation_year%></p>
+<p><%=  @movie.genre%></p>
+<p><%= @movie.actors.ordered_by_age.join(", ") %></p>
```

Comment:
Nice work adding `.join` here - because this is dealing with display logic, this can be called in the view!